scripts/test/runtests.sh

```diff
@@ -43,7 +46,8 @@ failCount=0
 
 for project in ${TestProjects[@]}
 do
-    ./corerun  "xunit.console.netcore.exe" "$project.dll" -xml "${project}-testResults.xml" -notrait category=failing
+    # corehost is on the PATH because it is in the stage 2 output
+    corehost "xunit.console.netcore.exe" "$project.dll" -xml "${project}-testResults.xml" -notrait category=failing
```

Comment:
I do think there may be a need to have `corehost` on the path when the dotnet sdk is installed. There are some scenarios in which a user may want to take a managed binary and run it (`corerun`-style). Anyway, either way is fine with me. I can probably switch it back to `corerun`. The Windows one was broken because it was using the stage0 `corerun` that was provided on the PATH, but I changed it to `corehost` first while trying to fix that. I should be able to revert to `corerun` and change the Windows script to ensure it is using the local `corerun` rather than any `corerun` that happens to be on the PATH.